### DISCUSSION TITLE
todo.txt: Marking a task as DONE with x

### DISCUSSION THREAD

**Discussion by jjaimealeman** [gsantner/markor#1476] [Q&A] [answered]
## General information
* App version: v2.80(124)
* Android version: Android 12 ()
* Device: Google Pixel 4a 5G

### Description
I mark a task as done, the editor places an 'x' at the front. i expected it to be "archived" and moved to the "archive.txt" file but this is not happening.

Am I doing it wrong?! 😕

**Comment by gsantner** [comment, marked as answer]
no this is how the todo.txt format is specified and how it works.

you may mark done and undone as often you wish. To move out all done tasks, use the archive button.

you might also find markor's project website useful.
